### ISSUE TITLE
720p50 Video is listed as 720p60

### ISSUE THREAD

**Issue by arthur4ik** [TeamNewPipe/NewPipe#4467]
Hi there
![Screenshot_20201009-190947](https://user-images.githubusercontent.com/12992757/95612057-06f9b400-0a63-11eb-9061-8a6066efbf33.png)

the original Video is a 720p50 Video but NewPipe shows 720p60.
https://www.youtube.com/watch?v=WoRVAnLvpaY

Is it possible to show the "statistics for nerds" like in the original YouTube App?

Best regards

**Comment by wb9688** [comment]
>the original Video is a 720p50 Video but NewPipe shows 720p60.

Yup, those have the same itag. In the future we'll actually parse the resolution and frame rate instead of assuming things based on the itag. That depends on fixing certain other things first though, so it'll probably take a while.

**Comment by Feuerswut** [comment]
Still open: remember refractor

**Comment by iampopovich** [comment]
available video for tests 
https://www.youtube.com/watch?v=S5nNLtPA5f0
https://www.youtube.com/watch?v=-HqB15g_sCA
